src/js/media-controller.js

```diff
@@ -1284,7 +1287,34 @@ const airplaySupported = !!window.WebKitPlaybackTargetAvailabilityEvent;
 const castSupported = !!window.chrome;
 const fullscreenEnabled = document[fullscreenApi.enabled];
 
-function serializeTimeRanges(timeRanges = []) {
+/** @type {TimeRanges} */
+const emptyTimeRanges = Object.freeze({
```

Comment:
copied from elements, thanks @cjpillsbury.
We don't need it in main, though, it won't hurt either.